### ISSUE TITLE
A Way To Undo Accidental Track Breaks

### ISSUE THREAD

**Issue by WebSoak** [TeamOpenIndustry/ImmersiveRailroading#1268]
Accidentally breaking a difficult to place track can be quite frustrating, something as easy as accidentally clicking left click and result in a difficult to place track requiring a replacement. (EX: Carefully placed switches, Elevated or Overhead Tracks, Bridges, Slopes, Custom Curves)

Benefits: Anyone who accidentally broke a difficult to place track

**Comment by tANDyeet** [comment]
yeah, that will be an awesome feature. It might work as track blueprint but reversed. So if you break the track, track blueprint appears
